public/assets/frontend/css/now-ui-kit.css

```diff
@@ -2089,7 +2089,6 @@ fieldset[disabled]
 
 .form-group .form-control,
 .input-group .form-control {
-  margin-top: 2px;
```

Comment:
This needs to be changed in the SASS file and then rebuilt - it's in `resources/sass/now-ui` , and then `make build-assets`